### PR TITLE
Initial file re-org & convert injector to class

### PR DESCRIPTION
**Pull request by cgewecke**
#32

### CODE REVIEW

**Comment by cgewecke** [comment]
From meeting (Feb 14): move `ui/` into `src`

File: scripts/localchain.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const ganache = require('ganache-cli');
 const pify = require('pify');
 const Web3 = require('web3');
 
-const Simple = require('./test/sources/pass/simple');
-const SimpleWithImport = require('./test/sources/pass/simpleWithImport')
+const Simple = require('./../test/sources/pass/simple');
+const SimpleWithImport = require('./../test/sources/pass/simpleWithImport')
```
Comment (edisinovcic) [inline scripts/localchain.js]: Should work without ./ at the begining

**Comment by cgewecke** [comment]
@edisinovcic This is rebased against `staging` now, and #53 is rebased as well - it should go into staging clean. 

But this is a weird intermediate PR - there are changes in #53 to some of the paths rewritten here because TS is getting written to dist, etc. 

I don't know if you'd prefer to just merge this into staging (or even close) and review #53 instead? It's up to you.

**Comment by cgewecke** [comment]
Rebased for #57 (Continuous deployment)

**Pull request merged**